sys-apps/groff/groff-1.22.4-r1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+MY_P="${P/_/.}"
+
+DESCRIPTION="Text formatter used for man pages"
+HOMEPAGE="https://www.gnu.org/software/groff/groff.html"
+SRC_URI="mirror://gnu/groff/${MY_P}.tar.gz
+	mirror://gnu-alpha/groff/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+[[ "${PV}" == *_rc* ]] || \
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="examples X"
+
+RDEPEND="
+	X? (
+		x11-libs/libX11
+		x11-libs/libXt
+		x11-libs/libXmu
+		x11-libs/libXaw
+		x11-libs/libSM
+		x11-libs/libICE
+	)"
+DEPEND="${RDEPEND}
+	dev-lang/perl"
+
+DOCS=( BUG-REPORT ChangeLog MORE.STUFF NEWS PROBLEMS PROJECTS README TODO )
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.19.2-man-unicode-dashes.patch #16108 #17580 #121502
+)
+
+src_prepare() {
+	default
+
+	# honor Gentoo's docdir
+	sed -i -e '/^docdir=/s/^/#/' \
+		Makefile.am \
+		|| die "failed to modify Makefile.am"
+
+	# Make sure we can cross-compile this puppy
+	if tc-is-cross-compiler ; then
+		eapply "${FILESDIR}"/${PN}-1.22.4-cross-compile.patch
```

Comment:
Conditional patches are usually a problem as they increase risk of collision with other patches in various YSE combinations.

Reply:
True, but this patch is more or less exactly what the original sed did,
adjusted for the current source's layout.